packages/longhorn/longhorn-1.2/package.yaml

```diff
@@ -1,11 +1,5 @@
 url: https://github.com/longhorn/charts.git
 subdirectory: charts/longhorn
-commit: d3ac3c2ee47cd9bd420b36cdef0d4e4fb24f9622
+commit: e52e41f9524b35d9804f8f66a70a66fad57be2b3
 version: 101.0.0
 doNotRelease: true
```

Comment:
Can we remove this `doNotRelease: true`

Reply:
Yes, I have removed`doNotRelease: true`.